Solutions/Tanium/Analytic Rules/TaniumThreatResponseAlerts.yaml

```diff
@@ -0,0 +1,34 @@
+id: dd9aa0ff-7ac1-4448-879c-e1a18d5890b4
+name: Tanium Threat Response Alerts
+description: 'Alerts from Tanium Thread Response (THR) that can be acted upon by Tanium Sentinel Playbook'
+severity: High
+queryFrequency: 5m
+queryPeriod: 6m
+triggerOperator: gt
+triggerThreshold: 0
+tactics: []
```

Comment:
Please also include a relevantTechniques field.

Reply:
Please include the field and leave it blank similar to tactics.

Reply:
added - thank you!